docs/source/projects.rst

```diff
@@ -294,19 +294,32 @@ The project document is stored in JSON format in the project root directory and
     >>> print(project.doc.hello)
     'world'
 
-The project data is stored in HDF5 format in the project root directory and can be used to store similar types of data to the job data.
+The project data is stored in HDF5 format in the project root directory. Although it can be used to store similar types of data as the job document, it is meant for storage of large, array-like or dictionary-like information.
```

Comment:
```suggestion
The project data is stored in HDF5 format in a file named ``signac_data.h5`` in the project root directory. Although it can be used to store similar types of data as the project document, it is meant for storage of large, array-like or dictionary-like information.
```